infra/aws/terraform/backend.tf

```diff
@@ -251,7 +251,7 @@ resource "aws_security_group_rule" "inbound" {
   from_port   = 80
   to_port     = 80
   protocol    = "tcp"
-  cidr_blocks = ["1.1.1.1/32"] # Network interface IPs of NLB
+  cidr_blocks = ["0.0.0.0/0"] # This could also be the network interface IP of the NLB
```

Comment:
I'd drop the comment about what it could be, and explain that we have NLB in use, not ALB so we can't restrict access to particular IP. What do you think?

Reply:
It's already stated at the beginning of that file, but it doesn't hurt to explain it anywhere we may see fit. What do you propose?

Although I don't like comments, this particular one helped me understand the real purpose of the rule.

Reply:
Great, I like it!